relayer/claim_request.go

```diff
@@ -8,14 +8,16 @@ import (
 
 	ethcommon "github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/ethclient"
+	logging "github.com/ipfs/go-log"
 
 	"github.com/athanorlabs/atomic-swap/coins"
 	contracts "github.com/athanorlabs/atomic-swap/ethereum"
 	"github.com/athanorlabs/atomic-swap/net/message"
 )
 
 const (
-	relayedClaimGas = 70000
+	relayedClaimGas   = 70000  // worst case gas usage for the claimRelayer swapFactory call
+	forwarderClaimGas = 195000 // worst case gas usage when using forwarder to claim
```

Comment:
how did you get this? unit tests?

Reply:
Yes, it is roughly 50% more than what I was sampling in unit tests:
```text
130507 * 1.5
195760.5
```
It's only used in this logic, where I was probably leaning on the high side:
```go
	txCost := new(big.Int).Mul(suggestedGasPrice, big.NewInt(forwarderClaimGas))
	if balance.Cmp(txCost) < 0 {
		return fmt.Errorf("balance %s ETH is under the minimum %s ETH to relay claim",
			coins.FmtWeiAsETH(balance), coins.FmtWeiAsETH(txCost))
	}
```
Let me know what you think the value should be.

Reply:
if `130507` was the actual gas value you got, i'd just put that or *slightly* higher (like 131000). the gas cost will always be the same unless the contract changes

Reply:
I reduced it from 195000 to 156000 (roughly 16% above measured gas units).